modules/default/calendar/calendar.js

```diff
@@ -40,7 +40,6 @@ Module.register("calendar", {
 		hideTime: false,
 		showTimeToday: false,
 		colored: false,
```

Comment:
the colored option is ignored now with your PR. You should add some fallback logic for that in the depreaction warning block, so that existing configs work like before.

Reply:
Done on latest commit.